examples/workflows/cockroach_basic_cluster/main.tf

```diff
@@ -0,0 +1,87 @@
+variable "cluster_name" {
+  type     = string
+  nullable = false
+}
+
+variable "sql_user_name" {
```

Comment:
Is it actually required to create a SQL user with Terraform? I cancel out of that all the time in the CC UI.

Reply:
It's probably not required, I just followed the pattern from other workflows.